modules/storages/app/forms/storages/admin/provider_type_select_form.rb

```diff
@@ -61,7 +61,8 @@ def default_select_list_options(storage)
         caption: nil,
         include_blank: false,
         required: true,
-        disabled: storage.persisted?
+        disabled: storage.persisted?,
+        input_width: :large
```

Comment:
Taken from the figma design, this should be smaller than the rest. So maybe `:medium` would be better here?

Reply:
updated to `:small` 👍🏾